pkg/trace/info/stats.go

```diff
@@ -288,6 +284,49 @@ func (s *SpansMalformed) String() string {
 	return mapToString(s.tagValues())
 }
 
+const (
+	maxAbsSamplingPriority = 10
+	numPriorityCounters    = maxAbsSamplingPriority*2 + 1
+)
+
+// samplingPriorityStats holds the sampling priority metrics that will be reported every 10s by the agent.
+type samplingPriorityStats struct {
+	// tracesPerSamplingPriority holds counters for each priority in position MaxAbsPriorityValue + priority.
+	tracesPerSamplingPriority [numPriorityCounters]int64
```

Comment:
```suggestion
	counts [numPriorityCounters]int64
```
I think that calling this `counts` will be more than enough to understand what it is, within its context.

Reply:
Agreed :) thank you